org-formation/720-client-vpn/_tasks.yaml

```diff
@@ -190,7 +190,12 @@ VpnAuthRoutes:
           - "aws-admins"
           - "scientists"
           - "sageit-users"
-
+      # org-sagebase-dnt-dev
+      orca-vpc:
+        CIDR: "10.255.24.0/24"
+        AccessGroups:
+          - "aws-admins"
+          - "scientists"
```

Comment:
Thanks @zaro0508.  Are `scientists` just the rest of Sage?

Reply:
would you prefer sage staff instead?

Reply:
Yes, "sage-staff" would be great!

Reply:
done, changed to allow all staff to access.